packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -147,6 +152,60 @@ export type BaseInputProps = InputLabelProps &
      * Displays the character counter under the input field
      */
     maxCharacters?: number;
+    /**
+     * alignement of the text inside input field
+     */
+    textAlign?: 'left' | 'center' | 'right';
+    /**
+     * If true, focuses the input field on load
+     *
+     * **Note:**
+     * Automatically focusing a form control can confuse visually-impaired people using screen-reading technology and people with cognitive impairments.
+     * When autofocus is assigned, screen-readers "teleport" their user to the form control without warning them beforehand.
+     */
+    autoFocus?: boolean;
+    /**
+     * determines what return key to show on keyboard on mobile devices/virtual keyboard
+     * **Note**: Few values are platform dependent and might not render on all the platforms
+     *
+     * `enter` is only available on web
+     * `previous` is only available on native android
+     */
+    keyboardReturnKeyType?: 'enter' | 'go' | 'done' | 'next' | 'previous' | 'search' | 'send';
+    /**
+     * **Web only**
+     *
+     * Hints browser to display an appropriate virtual keyboard
+     *
+     *
+     */
+    inputMode?: 'text' | 'search' | 'tel' | 'email' | 'url';
```

Comment:
Should we keep it as `telephone` since `autoCompleteSuggestionType` also has a value of `telephone`?

Reply:
I'd thought that while writing by that will just lead to unnecessary overhead because
1. it's web only property and its internal
2. if I put `telephone` then again I'll need to create a map of `tel` ➡️ `telephone`